hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -573,9 +676,15 @@ public Response getContainerMisMatchInsights() {
       throw new WebApplicationException(ex,
           Response.Status.INTERNAL_SERVER_ERROR);
     }
-    return Response.ok(containerDiscrepancyInfoList).build();
+    Map<String, Object> response = new HashMap<>();
+    response.put("prevKey", containerDiscrepancyInfoList.get(
```

Comment:
if list size is empty, means if no discrepancy, than this may throw exception

Reply:
Thanks for the comment @sumitagrawl I have added a check for it !